modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -44,17 +39,12 @@
     },
     "RequestFromArchive": {
       "Parameters": {
-        "cma": {
-          "event.$": "$",
```

Comment:
Wrong lambda

Reply:
Concur, this is the wrong lambda and should be reverted.

Reply:
reverted. I forgot the change was made only in extract_filepath for now